### ISSUE TITLE
Weird colors when using bg-dark class

### ISSUE THREAD

**Issue by Geczy** [picturepan2/spectre#301]
https://codepen.io/Geczy/pen/zEdvwj

Notice this weird lighter color on the side 

![image](https://user-images.githubusercontent.com/1036968/31050418-b4ca7dd4-a617-11e7-9f56-8e59393b1822.png)

**Comment by Geczy** [comment]
@picturepan2 is this project still active? I see many open issues, many open pull requests, all without responses for months

**Comment by picturepan2** [comment]
Yes. It is active.

Yan Zhu
________________________________

**Comment by picturepan2** [comment]
@Geczy You mean the background color?

**Comment by Geczy** [comment]
Yes the background color

**Comment by picturepan2** [comment]
@Geczy That is the current color. I plan to set it darker.

**Comment by Geczy** [comment]
Sorry you're misunderstanding the issue, let me try again one sec

**Comment by Geczy** [comment]
Strange, I can't reproduce the issue anymore, even though my screenshot shows the issue.

The issue was there are two background colors even though all I did was add `bg-dark`

![image](https://user-images.githubusercontent.com/1036968/31232518-171a8ef4-a9b8-11e7-81de-a2d2513e403d.png)
